core/frontend/src/FrontendHubAccess.ts

```diff
@@ -19,9 +19,5 @@ export interface IModelIdArg {
 export interface FrontendHubAccess {
   getLatestChangeset(arg: IModelIdArg): Promise<ChangesetIndexAndId>;
   getChangesetFromVersion(arg: IModelIdArg & { version: IModelVersion }): Promise<ChangesetIndexAndId>;
-  /**
```

Comment:
Why delete documentation entirely instead of adjusting for your change?

Reply:
re-added the doc, and ran extract-api

Reply:
Thanks. Please don't resolve other people's comments.